packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -84,6 +85,7 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
     return (
         <QueryClientProvider client={queryClient}>
             <CommerceApiContext.Provider value={apiClients}>{children}</CommerceApiContext.Provider>
+            <ReactQueryDevtools />
```

Comment:
Just wanted to confirm: do we need to do anything on our end to make sure this is included in dev bundle only?

Reply:
@adamraya I know it says that this library isn't included for production builds, but 💯 sure, can you verify this using the analyze build and share the results in this PR?

Reply:
@vmarta It's the default behaviour as detailed in the React Query Devtools docs: https://tanstack.com/query/v4/docs/devtools#:~:text=React%20Query%20Devtools%20are%20only%20included%20in%20bundles%20when

@bendvc Here are the results of the analyze build on `develop` and `add-react-query-devtools` branch for the `test-commerce-sdk-reack` package running `MOBIFY_ANALYZE=true npm run build`: 
![analyze-build](https://user-images.githubusercontent.com/6024102/186944959-cbd1347a-f354-47fe-b132-a851913290ff.png)

I also deployed a bundle confirming React Query Devtools is not shown in production builds: https://scaffold-pwa-test-env-2.mobify-storefront.com/